### DISCUSSION TITLE
cant' export to csv

### DISCUSSION THREAD

**Discussion by BerkeSoysal** [dbeaver/dbeaver#14011] [Q&A]
Hello,

Before I created a bug, I just wanted to be sure. When I write a script and right-click -> export -> export as csv I get **no database selected** error. When I add 'use blabla' to the beginning of the query, I get an error on the script. I am using dbeaver on macOS and using MariaDB.

**Comment by LonwoLonwo** [comment]
Hello @BerkeSoysal 
Please provide more steps. Screens or videos will be very helpful!
right-click on the query or right-click on resultSet?
End what DBeaver version do you use?

**Reply by BerkeSoysal** [reply]
Hi. 
-End what DBeaver version do you use?
Version 21.2.1.202109181446
-right-click on the query or right-click on resultSet?
Both.
Please provide more steps. Screens or videos will be very helpful!
I will ASAP.

**Reply by LonwoLonwo** [reply]
Is your SQL Editor bounded with some catalog?

![2021-09-27 12_44_55-Window](https://user-images.githubusercontent.com/45152336/134895199-f38a5897-7665-436f-8db3-6bf6446edd31.png)

**Reply by BerkeSoysal** [reply]
yes, it is.

**Reply by LonwoLonwo** [reply]
I can't reproduce the problem for now.
Clear steps and screens/videos will be very helpful.
